bower.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "ng-oauth2",
+  "description": "An oauth2 implementation for angular based applications",
   "version": "0.0.6",
+  "hompage": "https://github.com/el-davo/ng-oauth2",
   "license": "MIT",
   "main": [
     "dist/ng-oauth2.min.js"
   ],
   "dependencies": {
-    "angular": "1.3.x",
-    "angular-mocks": "1.3.x",
+    "angular": "^1.3.x",
+    "angular-mocks": "^1.3.x",
```

Comment:
No worries. It looks like it just needs a resolution in bower.json. Try deleting you bower folder and then bower install again. It should ask you for a resolution. Give it an anwer and put a ! after your answer to preserve it